HotTips/TipOfTheDayWindow.xaml.cs

```diff
@@ -282,10 +303,12 @@ private void LikeButton_Click(object sender, RoutedEventArgs e)
                 PopulateLikeFilledImage();
             }
             PopulateDislikeImage();
+            _tipHistoryManager.SaveTipStatus(currentTip);
```

Comment:
The algorithm is:
1. Update the tipsSeen in memory.
2. Persist to registry.
I can only see you doing #2 here.